### ISSUE TITLE
x/sys/windows: add missing file attributes

### ISSUE THREAD

**Issue by Kropekk** [golang/go#25979]
In [syscall library for Windows](https://golang.org/pkg/syscall/?GOOS=windows#pkg-constants)  ([version for amd64 architecture](https://golang.org/pkg/syscall/?GOOS=windows&GOARCH=amd64#pkg-constants) ) not all file attributes are present. 

There are some constants already defined (FILE_ATTRIBUTE_*), but I would expect to see [all of them](https://msdn.microsoft.com/pl-pl/library/windows/desktop/gg258117(v=vs.85).aspx) or at least all applicable to syscall.SetFileAttributes(...) function ([msdn docs for underlying function](https://msdn.microsoft.com/pl-pl/library/windows/desktop/aa365535(v=vs.85).aspx)).

 On the other hand, the remaining ones would be helpful in parsing syscall.GetFileAttributes(...) return value.

Summing up, I think that all FILE_ATTRIBUTE_* constants should be present in Windows-OS syscall library.

(BTW, I wanted to attach some labels to my issue, but I think it's not possible for a newcomer)

**Comment by ianlancetaylor** [comment]
We aren't going to add any more constants to the syscall package.  They should be added to the golang.org/x/sys/windows package instead.  See https://docs.google.com/document/d/1QXzI9I1pOfZPujQzxhyRy6EeHYTQitKKjHfpq0zpxZs/edit#heading=h.5nqylt8x3hka .
